frontend/components/RestaurantForm.vue

```diff
@@ -84,6 +85,8 @@
           type="tel"
```

Comment:
Code Smell?

Reply:
I didn't touch that line. Also, it's useful to get a numbers-keyboard in this field